src/policies.rs

```diff
@@ -119,11 +119,11 @@ pub fn verify_git_commits<G: Git, P: Gpg>(
         let exclusions = find_and_verify_override_tags(
             &git,
             &gpg,
-            &commits,
+            &all_commits,
             config.override_tags_required,
             &mut keyring,
         )?;
-        let commits = commits_to_verify_with_exclusions(
+        let commits_without_exclusions = commits_to_verify_with_exclusions(
```

Comment:
This is a very confusing name.  Are these:
 - The commits without the excluded commits (implies that signed commits might be excluded alone)?
 - The full commit range subtract the excluded commits?
 - The full commit range?
It's made especially confusing by the function name XD
How about something more explicit like: `commits_not_manually_verified`?